8.2 Install and setup automapper, 8.5 Move repository interface to business layer

sub13 opened the pull request:
I added and i set automapper up, i little refactored code, i fixed two bug in our program: 1) bug in getSprintById view; 2) bug in alltasksforteamView. If i've done something wrong please write down. I'll try to fix as soon as i can.
UPDATE1:
I Added Lazy loading into our project, I replaced all repositories to generic one repository, I removed business word, I added Imodel interface with method Update. And i set up tests for these changes

SergeyMokin reviewed (changes requested):
Required updates from call

sub13 commented:
UPDATE:
I Added Lazy loading into our project, I replaced all repositories to generic one repository, I removed business word, I added Imodel interface with method Update. And i set up tests for these changes

SergeyMokin commented on src/Teams.Data/Repository/IRepository.cs:
```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Teams.Data.Repository
```
Why it is not in Bussiness layer?

SergeyMokin reviewed (changes requested):
Move project dependencies back
